src/components/Static/Downloads/Downloads.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Template from '../../Template/Template';
+import Card from '../../Card/Card';
+import Button from '../../Button/Button';
+
+function Downloads() {
+  return (
+    <Template>
+      <h1 style={{ marginTop: '48px' }}>Downloads</h1>
+      <h2>Project Releases</h2>
+      <Card title={'Cortex Command Community Project'}>
+        <p style={{
+          marginBottom: '18px',
+        }}>This is where you can find the community's plans for Cortex Command's future. There'll be rebalancing, new weapons and units, and much more</p>
```

Comment:
Minor typo - there should be a period at the end. All the copy will probably be retouched at some point (though honestly, it's perfectly fine as is), so it's not a big deal, but I noticed it and figured I should mention it.